### ISSUE TITLE
My dashboard summary stats display dynamic data

### ISSUE THREAD

**Issue by csprayregen** [pieforproviders/pieforproviders#451]
## ✨ What new feature do you want to see?
The summary stats on the top of my dashboard display. dynamic data. This is currently blocked by #428 and #449. When these are done, we need to update this issue such that each stat on the dashboard has its own ticket (e.g. one for revenue, one for attendance, etc.)

## 🧶 Acceptance Criteria
To be added

## 🖍️ Implementation Details
Figma link: https://www.figma.com/file/uCveVQQnYcw3MsenD1PNcG/P4P-2.0?node-id=3582%3A15258

> **NOTE TO ENGINEERS**
> 
> Please ensure that you are running Google Lighthouse and/or WebAIM (Wave) on any UI elements in your pull request to check for accessibility issues.
> 
> If it is out of scope to address those accessibility issues in your pull request (because the issue is in a parent component and needs refactoring to work, for example), **you are responsible for noting those issues in your pull request and writing bug tickets to address the issues - make sure you link the new tickets in your pull request**

**Comment by katelovescode** [comment]
@cjhaddad We should talk about whether or not this gets rolled up on the front end or the backend sometime soon
